system/Autoloader/Autoloader.php

```diff
@@ -116,6 +128,12 @@ public function register()
 
 		// Now prepend another loader for the files in our class map.
 		spl_autoload_register([$this, 'loadClassmap'], true, true); // @phpstan-ignore-line
+
+		// Load our non-class files
+		foreach ($this->files as $file)
+		{
+			$this->includeFile($file);
```

Comment:
need to ensure the `$file` is string as well? 

```suggestion
                      if (is_string($file)) 
                      {
                          $this->includeFile($file);
                      }
```